locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-1471   
**Check Name:** SnakeYaml: Constructor Deserialization Remote Code Execution   
**Severity:** CRITICAL   
**Fixed Version:** 2.0   
**Description:** SnakeYaml's Constructor() class does not restrict types which can be instantiated during deserialization. Deserializing yaml content provided by an attacker can lead to remote code execution. We recommend using SnakeYaml's SafeConsturctor when parsing untrusted content to restrict deserialization.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-1471